metadata.tex

```diff
@@ -182,6 +185,9 @@
       }
       \end{varwidth}
     };
+    \begin{pgfonlayer}{background}
+      \draw [shade, blur shadow={shadow blur steps=5, shadow opacity=40}] (current bounding box.north west) rectangle (current bounding box.south east);
```

Comment:
Table has different drop shadow opacity, is this intentional?

Reply:
nop, fixed